src/battle_util.c

```diff
@@ -3231,12 +3208,17 @@ u8 AtkCanceller_UnableToUseMove(void)
         case CANCELLER_THAW: // move thawing
             if (gBattleMons[gBattlerAttacker].status1 & STATUS1_FREEZE)
             {
-                if (IsThawingMove(gBattlerAttacker, gCurrentMove))
+                if (gBattleMoves[gCurrentMove].flags & FLAG_THAW_USER)
                 {
-                    gBattleMons[gBattlerAttacker].status1 &= ~(STATUS1_FREEZE);
-                    BattleScriptPushCursor();
-                    gBattlescriptCurrInstr = BattleScript_MoveUsedUnfroze;
-                    gBattleCommunication[MULTISTRING_CHOOSER] = B_MSG_DEFROSTED_BY_MOVE;
+                    if (gBattleMoves[gCurrentMove].effect == EFFECT_BURN_UP && !IS_BATTLER_OF_TYPE(gBattlerAttacker, TYPE_FIRE))
+                        /* Do nothing */;
+                    else
```

Comment:
```suggestion
                    if (!(gBattleMoves[gCurrentMove].effect == EFFECT_BURN_UP && !IS_BATTLER_OF_TYPE(gBattlerAttacker, TYPE_FIRE)))
```

Reply:
Resolved conflicts and added the flag to Sizzly Slide. No idea what's up with the empty files, they're not even empty in my local. Could merge the request and revert it they're an issue?

Reply:
I tried fiddling around and it seems to be some sort of issue from when you first merged in latest commits because the .github versions of the files didn't existing in your first commit (they were in .travis at the time) and then the merge introduced them and for some reason when you merged, their file modes changed.

I might suggest remaking this PR to resolve the issue since I wouldn't like unnecessary changes being introduced.

Reply:
Sure, sounds good to me. Will close this and open a new PR.